### PR TITLE
This is a mobile version portfolio

### PR DESCRIPTION
**Pull request by oluwajuwon8**
The work section was added to my mobile version portfolio.

### CODE REVIEW

**Review by george-shammar — CHANGES_REQUESTED** [review]
Hi @oluwajuwon8 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project with failing linters.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -55,7 +54,140 @@ <h4>I'm Azeezat</h4>
...
+        ><img src="image/image.png" alt="" /></a
+      ><button class="see" type="submit">
+          See Project
+        </button>
+      <a href="#"
+        ><img src="image/image.png" alt="" /></a
+      ><button class="see" type="submit">
+          See Project
+        </button>
+    </div>
```
Comment (TedLivist) [inline index.html]: - [ ] Your entire **WORKS** section should be wrapped in a `<section>` tag

-  [ ] Your design also looks very different from what is on the template. Please, your UI should look as similar to the template as possible.

**Your design:**
aziz-des

**Template:**
chidi-backg

- [ ] The main container of the projects should have `display: grid`

File: index.html [hunk 2]
```diff
@@ -55,7 +54,140 @@ <h4>I'm Azeezat</h4>
...
+      <a href="#"
+        ><img src="image/image.png" alt="" /></a
+      ><button class="see" type="submit">
+          See Project
+        </button>
+      <a href="#"
+        ><img src="image/image.png" alt="" /></a
+      ><button class="see" type="submit">
+          See Project
+        </button>
```
Comment (TedLivist) [inline index.html]: - [ ] Please each project should be wrapped in a different container.
- [ ] Each project container should also be displayed with a flexbox.
- [ ] Your projects currently have no titles. Please each project should have a title like **'Profesional Art Printing Data'** as defined in the Figma template.
- [ ] The title of each project should be wrapped in a `hx` tag.
- [ ] Each of your projects has no description paragraph as defined in the Figma template. Please add a description wrapped in a `<p>` tag.
- [ ] There also seems to be an issue with indentation the closing tag of your `a` tags. Please fix it.

File: index.html [hunk 3]
```diff
@@ -55,7 +54,140 @@ <h4>I'm Azeezat</h4>
...
+    </div>
+    <div class="grid">
+      <ul class="grid-container">
+        <li class="grid-item">Css</li>
+        <li class="grid-item">Html</li>
+        <li class="grid-item">bootstrap</li>
+        <li class="grid-item">Ruby</li>
+      </ul>
+    </div>
+    <button id="button" type="submit">See Project</button>
```
Comment (TedLivist) [inline index.html:81]: - [ ] All your **'See Project'** buttons should have stylings for `:active`. Currently, you only have styling for `:hover`.

File: index.html [hunk 4]
```diff
@@ -55,7 +54,140 @@ <h4>I'm Azeezat</h4>
...
+    <section class="about">
+      <h2>About Me</h2>
+      <h3>
+        I'm a software Developer! I can help you build a product,feature or
+        website. Look through some of my work and Experience. if you like what
+        you see and have a project dont hesitate to contact me.
+      </h3>
+      <button id="Resume" type="submit">Get My Resume</button>
+
+      <div id="ground"><img src="Group2.png" alt=""/></div>
```
Comment (TedLivist) [inline index.html:127]: - [ ] Your background image should be set from the CSS, not `` tag. Please fix this.
- [ ] Also there seems to be an issue with your overall design. It is quite different from what is in the template. Please stick to the template as much as possible.

aziz-about

File: index.html [hunk 5]
```diff
@@ -55,7 +54,140 @@ <h4>I'm Azeezat</h4>
...
+    </div>
+    <br />
+    <br />
+    <section class="about">
+      <h2>About Me</h2>
+      <h3>
+        I'm a software Developer! I can help you build a product,feature or
+        website. Look through some of my work and Experience. if you like what
+        you see and have a project dont hesitate to contact me.
+      </h3>
```
Comment (TedLivist) [inline index.html]: - [ ] Please wrap this description in a `p` tag, not `hx` tag

File: index.html [hunk 6]
```diff
@@ -55,7 +54,140 @@ <h4>I'm Azeezat</h4>
...
+      <button id="Resume" type="submit">Get My Resume</button>
```
Comment (TedLivist) [inline index.html:125]: - [ ] The font of your button is quite large compared to what is on the template. Please reduce it.
- [ ] Your button should also have more horizontal padding as in the template.
- [ ] Add stylings for `:active` and `hover` for your button

btns-styles

File: index.html [hunk 7]
```diff
@@ -55,7 +54,140 @@ <h4>I'm Azeezat</h4>
...
+      <div class="skill">
+        Skills
+        <ul>
+          <li>Codekit</li>
+          <li>Github</li>
+          <li>Codepen</li>
+          <li>Gitlab</li>
+          <li>Terminal</li>
+        </ul>
+      </div>
```
Comment (TedLivist) [inline index.html:161]: - [ ] Please wrap your **'Languages'**, **'Frameworks'** and **'Skills'** categories in a `ul` with each of them in a `li` tag

langs

**Review by TedLivist — CHANGES_REQUESTED** [review]
Hi @oluwajuwon8,

Good job so far! You have done well setting up the linters.
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashbaord. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**